course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineScreen.kt

```diff
@@ -224,6 +227,10 @@ private fun CourseOutlineUI(
                 Box {
                     when (uiState) {
                         is CourseOutlineUIState.CourseData -> {
+                            if (uiState.courseStructure.blockData.isEmpty()) {
+                                NoContentScreen(noContentScreenType = NoContentScreenType.COURSE_OUTLINE)
+                                return@Surface
```

Comment:
Strange to se return inside compose

Reply:
`NoContentScreen` is a fullscreen error, so we don't want to show other content, so if the dates are empty return the flow, after showing the No content screen.

Reply:
I'm afraid that return will broke a [compose](https://developer.android.com/develop/ui/compose/mental-model) tree, can you please use else in the CourseOutlineUIState.CourseData